packages/toolpad-app/src/toolpad/AppEditor/BindingEditor.tsx

```diff
@@ -138,8 +138,8 @@ function JsExpressionPreview({ jsRuntime, input, globalScope }: JsExpressionPrev
 export interface EnvBindingEditorProps extends WithControlledProp<EnvAttrValue | null> {}
 
 export function EnvBindingEditor({ value, onChange }: EnvBindingEditorProps) {
-  const { envVarNames = [] } = useBindingEditorContext();
-
+  const context = useBindingEditorContext();
+  const env = Object.values(context.env ?? {});
```

Comment:
Let's still show the keys here, `Object.keys(env)`, not `Object.values(env)`. 
We can call the variable `envVarNames` like the previous name => `const envVarNames = Object.keys(env)`